src/components/Article.jsx

```diff
@@ -1 +1,35 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const postDate = date;
```

Comment:
we can use siple date without this variable. It takes 8 bit of data)))

Reply:
yeah, really😅